karma-dashboard.config.cjs

```diff
@@ -116,7 +118,7 @@ module.exports = function (config) {
     logLevel: config.LOG_INFO,
 
     // enable / disable watching file and executing tests whenever any file changes
-    autoWatch: true,
+    autoWatch: false,
```

Comment:
Is this necessary?

Reply:
A leftover from testing, can undo.

Although worth noting that re-running the whole test suite whenever any file changes is expensive.

Reply:
I normally would set `fit` or `fdescribe` on one test and keep rerunning until it works. Is this what controls that?

Reply:
Yep that's the one.

I will undo the change then to not break that workflow.